upgrade-instructions.md

```diff
@@ -35,6 +35,17 @@ Change list:
  - [Log4J2 Adoption](#log4j2-adoption)
  - [Drop Cassandra schema version prior version 5](#drop-cassandra-schema-version-prior-version-5)
  - [mailqueue.size.metricsEnabled now defaults to false](#mailqueuesizemetricsenabled-now-defaults-to-false)
+ - [Swift support has been dropped](#swift-support-has-been-dropped)
+
+### Swift support has been dropped
+
+Date ??/07/2020
+
+JIRA: https://issues.apache.org/jira/browse/JAMES-3028
+
+Concerned product: Distributed James
+
+OpenStack's Swift support has been dropped, please use Cassandra or AWS S3 instead.
```

Comment:
missing some rationals about this replacement (mainly because Swift also support S3 API)